kythe/typescript/indexer.ts

```diff
@@ -984,10 +985,15 @@ class Visitor {
           this.visit(element);
         }
         break;
+      case ts.SyntaxKind.ComputedPropertyName:
+        this.visit(decl.name.expression);
+        break;
+      // Nothing meaningful can be indexed about a literal expression.
+      case ts.SyntaxKind.StringLiteral:
+      case ts.SyntaxKind.NumericLiteral:
```

Comment:
Maybe you intended these for another change (?)

Reply:
Sure, I'll put this in another change.